### PR TITLE
feat: Support revoke refresh token to oidc logout function

### PR DESCRIPTION
**Pull request by mario-mui**
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by vinzscam — APPROVED** [review]

**Comment by mario-mui** [comment]
@vinzscam Ready to be merged？

**Comment by benjdlambert** [comment]
Wanna bring in @Rugvip here to see if this also makes sense before clicking the merge button :)

**Review by Rugvip — APPROVED** [review]
Yep makes sense, thank you! :tada:

**Pull request merged**
